### DISCUSSION TITLE
Why the Java QueueInfo Class is different with the HTTP Resopnse

### DISCUSSION THREAD

**Discussion by liamlull** [rabbitmq/hop#286] [Q&A]
Hi Support and Author
When I try to use the HOP to get the QueueInfo, But I found the QueueInfo class is different with the HTTP API response, Who Can tell me why, I found some propertites is not exist in the QueueInfo Class.
Like these  ("online", "open_files" and “reductions”)
![image](https://user-images.githubusercontent.com/38729399/160347277-6a2949fb-042f-4d39-91df-61fb0d2e34e6.png)
![image](https://user-images.githubusercontent.com/38729399/160347394-b5526930-d2af-4db2-a871-fecee1add8cb.png)

**Comment by michaelklishin** [comment]
Because no one needed them enough to contribute. This is open source software after all.

You are welcome to submit a PR that adds those properties.
